library/time.po

```diff
@@ -1503,13 +1505,13 @@ msgid ""
 "point, unaffected by frequency or time adjustments and not incremented while "
 "the system is asleep."
 msgstr ""
+"Horloge qui s'incrémente monotoniquement, suiviant le temps depuis un point "
+"arbitraire, non affecté par la fréquence ou les ajustements de temps et non "
+"incrémenté pendant que le système est endormi."
 
 #: ../Doc/library/time.rst:811
-#, fuzzy
 msgid ":ref:`Availability <availability>`: macOS 10.12 and newer."
-msgstr ""
-":ref:`Disponibilité <availability>`: Linux 2.6.28 et ultérieur, MacOS 10.12 "
-"et ultérieur."
+msgstr ":ref:`Disponibilité <availability>`: macOS 10.12 et ultérieur."
```

Comment:
```suggestion
msgstr ":ref:`Disponibilité <availability>` : macOS 10.12 et ultérieur."
```

/tmp/padpo_23dyc37u/library/time.po:1512 [     NBSP     ] There should be a non-breakable space before ":": between ### Disponibilité <availability> »### and ###: macOS 10.12 et ultérieur.###